### ISSUE TITLE
Inconsistent formatting for HTML files in 0.56.0

### ISSUE THREAD

**Issue by ymtszw** [prettier/prettier-atom#464]
- Windows 10, Atom 1.32.2
- prettier-atom 0.56.0

Let's say I have this HTML file, with some "bad" code styling in script part:
```html
<html>
  <head>
    <title>Prettier Test</title>
  </head>
  <body>
    <script type="text/javascript">
      const v = "1"; 
      console.log("prettify me!") ;
      const obj = { var:v};
    </script>
  </body>
</html>
```

Then I run prettier 1.15.2 from Windows Subsystem of Linux bash:
```sh
$ prettier src/static/test.html
<html>
  <head>
    <title>Prettier Test</title>
  </head>
  <body>
    <script type="text/javascript">
      const v = "1";
      console.log("prettify me!");
      const obj = { var: v };
    </script>
  </body>
</html>
⏎
```

On the other hand, same file, via prettier-atom (format on save):
```html
<html>
  <head>
    <title>Prettier Test</title>
  </head>
  <body>
    <script type="text/javascript">
    const v = "1"; console.log("prettify me!") ; const obj = { var:v};
</script>
  </body>
</html>

```

**Comment by ymtszw** [comment]
To add:
- I do not using eslint with prettier. 
  - Setting options: "ESLint Integration" disabled, "Stylelint Integration" disabled.
- No .eslintrc, .eslintignore files
- Other options:
  - "Only format if Prettier is found in your project's dependencies" disabled
  - "Only format if a Prettier config is found" disabled.
  - "Run Prettier Last" disabled.
- Prettier: Debug below
<details>
```
Atom version: 1.32.2
prettier-atom version: 0.56.0
prettier: bundled
prettier version: 1.15.2
prettier-eslint version: 8.8.2
prettier-atom configuration: {
  "formatOnSaveOptions": {
    "enabled": true,
    "showInStatusBar": true,
    "respectEslintignore": true,
    "excludedGlobs": [],
    "whitelistedGlobs": [],
    "isDisabledIfNotInPackageJson": false,
    "isDisabledIfNoConfigFile": false
  },
  "useEslint": false,
  "useStylelint": false,
  "prettierEslintOptions": {
    "prettierLast": false
  }
}
```
</details>

**Comment by ymtszw** [comment]
Also, it works consistently to CLI results for .js files. I find this behavior only in .html files.

**Comment by ymtszw** [comment]
Fix confirmed in 0.56.1. Thanks!
